packages/IntegrationTest/windows/integrationtest/TestModule.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <NativeModules.h>
+
+namespace IntegrationTest {
+
+using TestCompletedEventId = winrt::Microsoft::ReactNative::ReactNotificationId<void>;
+using TestPassedEventId = winrt::Microsoft::ReactNative::ReactNotificationId<bool>;
+
+REACT_MODULE(TestModule)
+struct TestModule {
+  static TestCompletedEventId TestCompletedEvent() noexcept;
+  static TestPassedEventId TestPassedEvent() noexcept;
+
+  REACT_INIT(Init)
+  void Init(const winrt::Microsoft::ReactNative::ReactContext &ctx) noexcept;
+
+  REACT_METHOD(markTestCompleted)
+  void markTestCompleted() noexcept;
+
+  REACT_METHOD(markTestPassed)
+  void markTestPassed(bool passed) noexcept;
```

Comment:
(nit)

Use Pascal case. The `REACT_METHOD` macro allows having different JS and C++ method names.

Reply:
I've seen this go both ways. E.g. built-in ClipboarModule, AlertModule, DevSettingsModule, and AccessibilityInfoModule just keep lowercase, but others like LogBoxModule have methods that remap capitalization.

My own preference is to use lowercase instead of remapping, since it removes a second source of truth if naming ever needs to change.